### PR TITLE
fix: Bug in JDTCommentBuilder

### PR DESCRIPTION
**Pull request by Egor18**
Hi! I've found some funny bug in JDTCommentBuilder. Model building fails with the following exception:
```
spoon.SpoonException: Unexpected next line after last line
	at spoon.support.compiler.jdt.JDTCommentBuilder.cleanComment(JDTCommentBuilder.java:595)
	at spoon.support.compiler.jdt.JDTCommentBuilder.getCommentContent(JDTCommentBuilder.java:557)
	at spoon.support.compiler.jdt.JDTCommentBuilder.buildComment(JDTCommentBuilder.java:143)
	at spoon.support.compiler.jdt.JDTCommentBuilder.build(JDTCommentBuilder.java:108)
```
on this code:
```
class Test {
     /*/
     Hello!
     */
}
```
The problem is that `*/` from `/*/` is considered as the end of the comment. This PR provides possible solution and some additional tests.

### CODE REVIEW

File: src/test/java/spoon/test/comment/testclasses/WildComments.java [hunk 1]
```diff
@@ -115,5 +115,24 @@
...
+		comment and slash
+		/*/
+
+		/*/
+		slash and comment and slash
+		/*/
+
+		//*/
+
+		/*/*/
```
Comment (pvojtechovsky) [inline src/test/java/spoon/test/comment/testclasses/WildComments.java]: Could you please move all these new comment examples above (to line >116). This test expects pairs of tested comment example and expected string content of the comment.

Comment (Egor18) [reply src/test/java/spoon/test/comment/testclasses/WildComments.java]: Should I create a new file for them?

Comment (pvojtechovsky) [reply src/test/java/spoon/test/comment/testclasses/WildComments.java]: > new file

no, just move them up and add expected content of the comment text. For example:
convert this
```java
/*/
slash and comment
*/
```
to
```java
/*/
slash and comment
*/
"/\nslash and comment"
```

**Comment by pvojtechovsky** [comment]
Thank You 👍

**Pull request merged**